# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../lua/ar.patch
# Copyright (C) 2009 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./src/Makefile.orig	2009-05-12 23:25:09.000000000 +0000
+++ ./src/Makefile	2009-05-12 23:25:41.000000000 +0000
@@ -9,7 +9,7 @@
 
 CC= gcc
 CFLAGS= -O2 -Wall $(MYCFLAGS)
-AR= ar rcu
+AR= ar
 RANLIB= ranlib
 RM= rm -f
 LIBS= -lm $(MYLIBS)
@@ -48,7 +48,7 @@
 a:	$(ALL_A)
 
 $(LUA_A): $(CORE_O) $(LIB_O)
-	$(AR) $@ $?
+	$(AR) rcu $@ $?
 	$(RANLIB) $@
 
 $(LUA_T): $(LUA_O) $(LUA_A)
